ROOTFS/etc/waggle/sanity/warning/check_hanwha_camera.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+import urllib.error, urllib.request
+import time
+
+def read_camera_manifest(path='/etc/waggle/node_manifest.json'):
+    with open(path, 'r') as file:
+        manifest = json.loads(file.read())
+        if 'cameras' not in manifest:
+            raise Exception(f'No cameras in the manifest: {path}')
+        else:
+            for camera in manifest['cameras']:
+                if camera['present'] == True:
+                    yield camera
+
+
+def http_request(url, user='waggle', pw='Waggle1@34', timeout=2):
```

Comment:
Is this timeout (which I assume is 2 seconds) enough time for a "slow" camera?  I know that a longer timeout makes the test take longer for the IP addresses that do *not* have a camera.  But I would rather be conservative and okay with this test taking longer.

Remember it is only run once every hour (right now).  So, its okay if this test takes a minute or so.

So maybe a 5 second timeout is more conservative??

Reply:
Sounds good! Changed it to 5 seconds.